tee-worker/docker/lit-ts-api-package-build.yml

```diff
@@ -13,11 +13,13 @@ services:
             litentry-node:
                 condition: service_healthy
             litentry-worker-1:
-                condition: service_healthy
+                # using +service_started+ over +service_healthy+ since worker runs successfully but can not connect to parachain
+                # as requires additional pre-setup for parachain image which built in production mode
+                # for generating types there is no need for fully workable interaction between worker and parachain
+                condition: service_started
```

Comment:
Does it mean we can get the data even tho enclave is not registered on the parachain? (I assume so

Reply:
yes, that's correct. I have one concern though, what if not registred enclave generate types which are different in case it was registred 🤔